service/service.py

```diff
@@ -175,7 +175,13 @@ def query_wishlist():
     customer_id = request.args.get('customer_id')
     name = request.args.get('name')
     wishlist = []
-    if wishlist_id:
+    if name and customer_id:
```

Comment:
there should be one more case where name, customer_id and wishlist_id are provided.

Writing extensive conditionals should work for now. However, there should be a better way to do this. I'm not sure if there is any

Reply:
@wjduand Check if this helps https://stackoverflow.com/a/31063911

Reply:
> @wjduand Check if this helps https://stackoverflow.com/a/31063911

It seems like a smart way, but since it's defined as classmethod I don't think it can be passed as a list.